### PR TITLE
feat(loaders): add skeleton component

### PR DESCRIPTION
**Pull request by rossmoody**
## Description

This PR adds the Skeleton loader component page to the website. 

## Checklist

- [ ] :ok_hand: website updates are Garden Designer approved (add the designer as a reviewer)
- [ ] :black_nib: copy updates are approved (add the content strategist as a reviewer)
- [ ] :wheelchair: analyzed via [axe](https://www.deque.com/axe/) and evaluated using VoiceOver
- [ ] :zap: audited via [web.dev](https://web.dev/measure/) for performance and SEO
- [ ] :memo: tested in Chrome, Firefox, Safari, Edge, and IE11

### CODE REVIEW

File: src/pages/components/skeleton.mdx [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+title: Skeleton
+description: >
+  A Skeleton loader estimates the amount of space needed by acting as a
```
Comment (vsrixyz) [inline src/pages/components/skeleton.mdx:4]: A Skeleton loader shows users a blank version of a page into which content is gradually loaded.

Comment (rossmoody) [reply src/pages/components/skeleton.mdx:4]: I'm thinking this update might remove some needed key words for introducing a Skeleton. Their big benefit is they approximate space for content during page load to make for a more pleasant loading experience (not sure it's fair to say it shows a blank page).

Comment (jzempel) [reply src/pages/components/skeleton.mdx:4]: Also, sometimes skeletons are used for dynamic loading of content that changes within a rendered page.

Comment (vsrixyz) [reply src/pages/components/skeleton.mdx:4]: How about:

A Skeleton loader shows users a blank version of a page or section of a page into which content is gradually loaded. It provides a visual estimate of the space needed.

**Pull request merged**